dosomething-qa/fastly-frontend/projectpages_recv.vcl

```diff
@@ -0,0 +1,4 @@
+# Catch any */project/* page requests.
+if (req.url ~ "(?i)^\/((us|mx|br)\/)?project\/")
+  error 779 "Not Found";
```

Comment:
Whoop, these have a syntax error in them:

```
* fastly_service_v1.frontend-qa: [ERR] Invalid configuration for Fastly Service (5I35is1gX0tbds1VfNmBIb): Syntax error: Expected `{`, got `error`
at: (input Line 657 Pos 3)
  error 779 "Not Found";
--#####-----------------
```

We need to add an opening `{` on the if conditional above!

Reply:
doh!